terraform/aws/components/nhais/variables.tf

```diff
@@ -78,7 +78,19 @@ variable "mq_broker_name" {
 variable "nhais_mongo_options" {
   type = string
   description = "Options for Mongo"
-  default = "replicaSet=rs0&readPreference=secondaryPreferred&retryWrites=false"
+  default = "replicaSet=rs0&readPreference=secondaryPreferred&retryWrites=false&tls=true"
+}
+
+variable "nhais_mongo_documentdb_tls_enabled" {
+  type = string
+  description = "Enable TLS on AWS Document DB"
+  default = "true"
```

Comment:
same as with base, I guess default is false, the actual value set in etc/ files.

Reply:
default is false, we set true only for exemplar

Reply:
It should be only enabled on V&P now, once testing is completed will apply to PTL (exemplar)